setup.cfg

```diff
@@ -93,6 +73,10 @@ ignore =
     E203,
     # black takes care of spaces after commas
     E231,
-per-file-ignores =
-    src/plone/base/interfaces/__init__.py:F401
-    src/plone/base/permissions.py:F401,E402
```

Comment:
Keep these lines so you don't have to add `  # noqa: F401` all over the place 😅

Reply:
This got overridden. `config-package.py` should not do that!